.github/auto-assign-config.yml

```diff
@@ -0,0 +1,7 @@
+addReviewers: true
+
+addAssignees: author
+
+reviewers:
+  - RonWang
+  - kakhaUrigashvili
```

Comment:
Will it be possible that the author will review his own PR? :)

Reply:
no. I tested on this PR.